server/services/withings/lib/withings.deleteDevices.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @description Delete all devices (and variables) of withings integration.
+ * @returns {Promise} Resolve with succes result.
+ * @example
+ * withings.deleteDevices();
+ */
+async function deleteDevices() {
+  await this.gladys.device.destroyByServiceId(this.serviceId);
+  return { success: true };
```

Comment:
no need to return {success: true}, it's not even used after

Reply:
i fix it